src/components/Themes/ui/PreviewTab/TasksPreview/TasksPreview.tsx

```diff
@@ -0,0 +1,856 @@
+                                spacing={{p: 5}}
+                                overflow="auto"
+                                style={{backgroundColor: contrastBackgroundColor}}
+                            >
+                                <Card
+                                    overflow="auto"
+                                    view="raised"
+                                    style={{
+                                        flexGrow: 1,
+                                        flexBasis: '50%',
+                                        backgroundColor: 'var(--g-color-base-background)',
+                                        border: 'none',
+                                    }}
+                                >
+                                    <Flex direction="column" overflow="auto" height="100%">
+                                        <Flex
+                                            spacing={{mt: 4, mx: 5}}
+                                            alignItems="center"
+                                            justifyContent="space-between"
+                                        >
+                                            <Flex alignItems="center">
+                                                <Text variant="subheader-2">Today</Text>
+                                                <Button view="flat-secondary">
+                                                    <Icon data={Calendar} />
+                                                </Button>
+                                            </Flex>
+                                            <Flex gap={2}>
+                                                <Button view="flat-secondary">
+                                                    <Icon data={ArrowUpArrowDown} />
+                                                    Sort
+                                                </Button>
+                                                <div>
+                                                    <Button
+                                                        view="action"
+                                                        pin="round-brick"
+                                                        style={{marginRight: '1px'}}
+                                                    >
+                                                        <Icon data={Plus} />
+                                                        Add Task
+                                                    </Button>
+                                                    <Button view="action" pin="brick-round">
+                                                        <Icon data={ChevronDown} />
+                                                    </Button>
+                                                </div>
+                                            </Flex>
+                                        </Flex>
+                                        <TabList value="active">
```

Comment:
Need to add some padding

<img width="315" alt="image" src="https://github.com/user-attachments/assets/03ae1316-bfa1-4629-bc81-487329d1f03f" />